src/load_dico.py

```diff
@@ -1,7 +1,35 @@
-import json
+from googleapiclient.discovery import build
+from google_auth_oauthlib.flow import InstalledAppFlow
+from google.auth.transport.requests import Request
+from google.oauth2.credentials import Credentials
 
-def load_dico(json_name):
-    with open(json_name) as json_file:
-        data = json.load(json_file)
-    dico = {line['word']: line['relations'] for line in data}
-    return dico
\ No newline at end of file
+def load_dico(manulex_spreadsheet_id, refresh_token, client_id, client_secret):
+    credentials = Credentials(
+        None,
+        refresh_token=refresh_token,
+        client_id=client_id,
+        client_secret=client_secret,
+        scopes=['https://www.googleapis.com/auth/spreadsheets.readonly'],
+        token_uri="https://accounts.google.com/o/oauth2/token"
+    )
+    service = build('sheets', 'v4', credentials=credentials)
+
+    sheet = service.spreadsheets()
+    result = sheet.values().get(spreadsheetId=manulex_spreadsheet_id, range='dico manulex!A1:E').execute()
+    values = result.get('values', [])
+
+    lines = []
+    def format_line(line):
+        word, phonetic, blending, relations = line
+
+        return ({
+            'word': word,
+            'phonetic': phonetic,
+            'blending': blending.split('-'),
```

Comment:
Well I was kinda surprised to see no error whatsoever in the current spreadsheet, and it made me thinking that if errors are very rare it may be better to not hide them, at least not until we have a proper error reporting tool in place. But I'm not dogmatic on this, either way suits me!